avocado/plugins/multiplex.py

```diff
@@ -82,29 +82,15 @@ def run(self, args):
             sys.exit(exit_codes.AVOCADO_ALL_OK)
 
         log.info('Variants generated:')
-        for (index, tpl) in enumerate(variants.variants):
-            if not args.mux_debug:
-                paths = ', '.join([x.path for x in tpl])
-            else:
-                color = output.TERM_SUPPORT.LOWLIGHT
-                cend = output.TERM_SUPPORT.ENDC
-                paths = ', '.join(["%s%s@%s%s" % (_.name, color,
-                                                  getattr(_, 'yaml',
-                                                          "Unknown"),
-                                                  cend)
-                                   for _ in tpl])
-            log.debug('%sVariant %s:    %s', '\n' if args.contents else '',
-                      index + 1, paths)
-            if args.contents:
-                env = set()
-                for node in tpl:
-                    for key, value in node.environment.iteritems():
-                        origin = node.environment_origin[key].path
-                        env.add(("%s:%s" % (origin, key), str(value)))
-                if not env:
-                    continue
-                fmt = '    %%-%ds => %%s' % max([len(_[0]) for _ in env])
-                for record in sorted(env):
-                    log.debug(fmt, *record)
+        if args.mux_debug:
+            # In this version `avocado_variants.debug` is not set properly,
+            # let's force-enable it before calling to_str to
+            # get the expected results.
+            args.avocado_variants.debug = True
+        use_utf8 = settings.get_value("runner.output", "utf8",
```

Comment:
True for line 90, it is not needed as `variants` does not support utf8. On line 79 it's used so I'm keeping it.